features/FEATURE_LWIP/lwip-interface/ppp_lwip.cpp

```diff
@@ -18,6 +18,14 @@
 #include "platform/FileHandle.h"
 #include "platform/mbed_poll.h"
 #include "events/EventQueue.h"
+#if defined(FEATURE_COMMON_PAL)
+#include "mbed_trace.h"
+#define TRACE_GROUP "LPPP"
+#else
+#define tr_debug(...) (void(0)) //dummies if feature common pal is not added
```

Comment:
why ATParser uses `debug_if` and this file `tr_debug` ? They should both be the same. Conditional `FEATURE_COMMON_PAL` is not acceptable